src/item.cpp

```diff
@@ -91,6 +91,7 @@ std::pair<std::string, offset_type> Item::getDirectAccessInformation() const
   }
   auto range = first_part->first;
   auto part = first_part->second;
-  auto local_offset = full_offset - range.min;
-  return std::make_pair(part->filename(), offset_type(local_offset));
+  const offset_type local_offset(full_offset - range.min);
+  const offset_type offset = local_offset + m_file->getArchiveStartOffset().v;
```

Comment:
Shouldn't we add the `archiveStartOffset` to the `full_offset` before we search for the part ?

We have a `archiveStartOffset` different of zero only when we open a embedded archive (so only one part) for now but we may have in the future some kind of opening of embedded archive in a split file.

Reply:
Makes sense